profile/README.md

```diff
@@ -1,24 +1,25 @@
 # Cosmos
 
-The Cosmos org is collection of teams working on decentralized technology.
+A collection of teams working on decentralized technology for the internet of blockchains - The Interchain.
 
-## Quick Start
+| [Developer Portal](https://tutorials.cosmos.network) - Quick Start | -  | [Interchain Discord](https://discord.com/invite/interchain) - For Support and Community |
```

Comment:
Don't we want people to ask support question on GH discussion instead?
Who is actually looking at discord forum except Reece 😅

Reply:
It's a good point 😋
I think for now at least, Discord is still the place with the highest support question volume, even with effectively just @Reecepbcups in the trenches - and we have some plans for helping Reece out.
We can also have further conversations on this, but for now, I still think Discord is the place to pass general support to, particularly for newer devs.

Reply:
I'd argue that it is easier for us to check (I've muted discord personally) and at least the answers are indexed. So it avoids answering 15 times the same thing and gives better SEO.

Reply:
I've been manually indexing common questions already from discord. So we can put these into guides or deliver on github for better SEO. There are still a lot more over the past year I have missed, but it's a good start

![image](https://github.com/cosmos/.github/assets/31943163/cc45bb72-f1c8-427a-82e8-807ffd512cc6)